jpa/pom.xml

```diff
@@ -39,31 +39,9 @@
 
     <modules>
         <module>hibernate6</module>
+        <module>subsystem</module>
+        <module>spi</module>
+        <module>eclipselink</module>
     </modules>
 
-    <profiles>
-        <profile>
-            <id>ee8.build.profile</id>
-            <activation>
-                <property>
-                    <name>!skip.ee8</name>
-                </property>
-            </activation>
-            <modules>
-                <!-- TODO once moved to jakarta.* move to main module set-->
-                <module>subsystem</module>
-                <!-- TODO once moved to jakarta.* move to main module set-->
-                <module>spi</module>
-                <!-- TODO remove-->
-                <module>hibernate5</module>
-                <!-- TODO remove-->
-                <module>hibernate5_3</module>
-                <!-- TODO once moved to jakarta.* move to main module set-->
-                <module>eclipselink</module>
-                <!-- TODO once moved to jakarta.* move to main module set-->
-                <module>openjpa</module>
-            </modules>
-        </profile>
-    </profiles>
```

Comment:
@scottmarlow This profile should be restored and subsystem, spi, eclipselink moved back to it. The others can be removed.

Getting rid of this profile would be done as part of converting the subsystem, spi, eclipselink source code to what's currently used to build ee-9/source-transform/jpa/xxx. Until then this profile allows us to turn off pointless builds of the javax subsystem, spi, eclipselink modules (e.g. to save CI resources).

Reply:
Should we eliminate the `ee-9/source-transform/jpa/eclipselink/pom.xml, ee-9/source-transform/jpa/spi/pom.xml, ee-9/source-transform/jpa/subsystem/pom.xml (removing them from ee-9/source-transform/pom.xml + ee-9/common/src/main/resources/license/preview-feature-pack-common-licenses.xml) as part of this pull request?  As well as switch to jakarta namespace within the wildfly/jpa tree?

Or should we do that separately from this pr?